deb/jjs-oneclick

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
```

Comment:
Set -e ?

Reply:
I think it shouldn't be set, so e.g. database setup will still run if sysroot setup fails.

Reply:
Is there situation when creating  sysroot failed for legitimate reason, so we should continue setup?